app/views/HasCorrectGoodsView.scala.html

```diff
@@ -33,7 +33,7 @@ <h1 class="govuk-heading-l">@messages("hasCorrectGoods.heading", commodity.commo
     <p class="govuk-body">@messages("hasCorrectGoods.p1", commodity.description)</p>
 
     <p class="govuk-body">
-        <a href="https://www.trade-tariff.service.gov.uk/find_commodity" class="govuk-link" >@messages("hasCorrectGoods.p2.linkText")</a>.
+        <a href="https://www.trade-tariff.service.gov.uk/search?q=@commodity.commodityCode" class="govuk-link" >@messages("hasCorrectGoods.p2.linkText")</a>.
```

Comment:
In three amigos, Simon said the url should be https://www.trade-tariff.service.gov.uk/xi/commodities/commodityCode ie., https://www.trade-tariff.service.gov.uk/xi/commodities/0702000007